### PR TITLE
remove minute test

### PR DESCRIPTION
**Pull request by jamesgorrie**
Remove the minute test - we have no stats as it was a 0% test to check viability.

sorry @NataliaLKB 
cc @akash1810

### CODE REVIEW

**Comment by akash1810** [comment]
why so early?

**Comment by jamesgorrie** [comment]
It's been running for a month.
Best we decide what to do with it, We can add it back if we need it.

File: static/src/javascripts/projects/common/modules/experiments/ab.js [hunk 1]
```diff
@@ -61,10 +59,14 @@ define([
         new LiveBlogChromeNotificationsInternal(),
         new LiveBlogChromeNotificationsProd(),
         new LoyalAdblockingSurvey(),
+<<<<<<< HEAD
```
Comment (akash1810) [inline static/src/javascripts/projects/common/modules/experiments/ab.js]: the merge didn't work properly :'(

**Comment by akash1810** [comment]
other than a bad merge, 👍

**Comment by akash1810** [comment]
👍

**Pull request merged**